starter-code/pizza.js

```diff
@@ -1 +1,99 @@
-// Write your Pizza Builder JavaScript in this file.
+function calculatePrice(){
+  var totalPrice = 0;
+  var arrayAllElements = [];
+  $("aside > ul li").each(function(index,element ) {
+    arrayAllElements.push(element);
+  });
+  var arrayFiltered = arrayAllElements.filter(function(element){
+    return element.style.display !== 'none';
+  });
+
+  for(var i = 0;i<arrayFiltered.length;i++){
+    var text = arrayFiltered[i].innerHTML;
+    totalPrice += parseInt(text[1]);
+  }
+  totalPrice += 10;
+  totalPrice = "$"+totalPrice.toString();
+  return totalPrice;
+}
+
+$(document).ready(function(){
+  $("aside > ul li:contains('white sauce')").toggle();
+  $("aside > ul li:contains('gluten-free crust')").toggle();
+
+  var price = $("strong");
+
+
+  var list = $("aside > ul ");
+  var buttonPepperonni = $(".btn-pepperonni");
+  var pepperonni = $(".pep");
+
+
+  buttonPepperonni.on("click",function(){
+    pepperonni.toggle();
+    buttonPepperonni.toggleClass("active");
+    $("aside > ul li:contains('pepperonni')").toggle();
+    price.text(calculatePrice());
+
+
+
+
+  });
+  var pepper =$(".green-pepper");
+  var buttonPepper = $(".btn-green-peppers");
+
+
```

Comment:
The event listeners are pretty much similar, try to put them in one single function for the next time